src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -6,10 +6,13 @@
  */
 public class Bulldozer extends Machine {
     public Bulldozer() {
+        setName(getName());
```

Comment:
The no-args constructor is setting the name of the Bulldozer to its current name which is not initialized yet. This logic doesn't seem to serve any purpose and could potentially be removed unless there's a specific reason for this implementation.